### PR TITLE
add community representatives to front page

### PR DESCRIPTION
**Pull request by SethTisue**
I don't know how to preview the site locally (that's #38), so
perhaps my HTML is mangled -- someone should check that before
merging

partially addresses #56

/cc @bvenners @tpolecat

### CODE REVIEW

**Comment by julienrf** [comment]
@SethTisue I use the following `docker-compose.yml` file and then I can render the site locally with `docker-compose up`, in case that helps:

~~~ yaml
version: '2'

services:
  jekyll:
    image: bretfisher/jekyll-serve
    volumes:
      - .:/site
    ports:
      - '4000:4000'
~~~

I’ll include it into the repository in case that’s useful.

**Comment by julienrf** [comment]
Screenshot from 2020-01-29 09-28-28

File: _layouts/frontpage.html [hunk 1]
```diff
@@ -384,6 +384,12 @@ <h2>Brought to you by</h2>
               -->
             </div>
             <div class="col-md-12 text-center">
+              <span class="subtitle col-sm-12">Community representatives:</span>
+              <div class="text-center back-level clearfix">
+                <a href="https://github.com/bvenners" class="col-md-4 col-xs-12" target="_blank">Bill Venners</a>
```
Comment (julienrf) [inline _layouts/frontpage.html]:
```suggestion
                <a href="https://github.com/bvenners" class="col-md-6 col-xs-12" target="_blank">Bill Venners</a>
```

File: _layouts/frontpage.html [hunk 2]
```diff
@@ -384,6 +384,12 @@ <h2>Brought to you by</h2>
...
+                <a href="https://github.com/tpolecat" class="col-md-4 col-xs-12" target="_blank">Rob Norris</a>
```
Comment (julienrf) [inline _layouts/frontpage.html]:
```suggestion
                <a href="https://github.com/tpolecat" class="col-md-6 col-xs-12" target="_blank">Rob Norris</a>
```

Comment (julienrf) [reply _layouts/frontpage.html]: This is how it looks like with those changes:

Screenshot from 2020-01-29 09-32-25

**Comment by SethTisue** [comment]
Thanks Julien! I added your suggested changes.

**Pull request merged**